core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksGPUTests.scala

```diff
@@ -11,11 +11,7 @@ import java.io.File
 import scala.collection.mutable.ListBuffer
 
 class DatabricksGPUTests extends DatabricksTestHelper {
-  val horovodInstallationScript: File = FileUtilities.join(
-    BuildInfo.baseDirectory.getParent, "deep-learning",
-    "src", "main", "python", "horovod_installation.sh").getCanonicalFile
-  uploadFileToDBFS(horovodInstallationScript, "/FileStore/horovod-fix-commit/horovod_installation.sh")
-  val clusterId: String = createClusterInPool(GPUClusterName, AdbGpuRuntime, 2, GpuPoolId, GPUInitScripts)
+  val clusterId: String = createClusterInPool(GPUClusterName, AdbGpuRuntime, 2, GpuPoolId, "[]")
```

Comment:
nit: Magic string "[]". Maybe this should just be a default value for createClusterInPool. Seems error-prone but not caused by this commit

Reply:
As we are comparing with master (spark32), this came as part spark33 changes: https://github.com/microsoft/SynapseML/commit/bab43c8b2bfd0e70e941287c03760888f51700d7#diff-10e2de13d08f0b488f4f20c5418631420d75dcad1aa269cbf10134ffe300aed0

@JessicaXYWang , can you please respond to this as you made spark33 changes

Reply:
The changes are made because GPUInitScripts is used to install horovod in databricks 3.2 pool. Since horovod is preinstalled for spark33 pool, we do not need GPUInitScript. I agree "[]" could be the default value.

Reply:
https://github.com/KeerthiYandaOS/SynapseML/pull/3

Reply:
Thank you @JessicaXYWang , Added default values here: https://github.com/microsoft/SynapseML/pull/2052/commits/c5ae6415e24265292c3af5b6186c48b280790e14